src/identity.js

```diff
@@ -1899,4 +1803,62 @@ export default function Identity(mpInstance) {
         }
         return userAttributeChangeEvent;
     };
+
+    this.reinitForwardersOnUserChange = function(prevUser, newUser) {
+        if (didUserChange(prevUser, newUser)) {
+            mpInstance._Forwarders.initForwarders(
+                newUser.getUserIdentities().userIdentities,
+                mpInstance._APIClient.prepareForwardingStats
+            );
+        }
+    };
+
+    this.setForwarderCallbacks = function(user, method) {
+        // https://go.mparticle.com/work/SQDSDKS-6036
+        mpInstance._Forwarders.setForwarderUserIdentities(
+            user.getUserIdentities().userIdentities
+        );
+        mpInstance._Forwarders.setForwarderOnIdentityComplete(user, method);
+        mpInstance._Forwarders.setForwarderOnUserIdentified(user);
+    };
+}
+
+function didUserChange(prevUser, newUser) {
+    return (
+        !prevUser ||
+        newUser.getMPID() !== prevUser.getMPID() ||
+        prevUser.isLoggedIn() !== newUser.isLoggedIn()
+    );
+}
+
+// https://go.mparticle.com/work/SQDSDKS-6359
+function tryOnUserAlias(previousUser, newUser, identityApiData, logger) {
```

Comment:
I don't think this needs to be exported and tested since we are going to deprecate